core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,16 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
+    .oppia-wide-panel-content {
+      margin-bottom: 240px;
+    }
     @media (max-width: 768px) {
       .oppia-footer {
         position: relative;
       }
+      .oppia-wide-panel-content {
```

Comment:
Hi @llKCll, this looks pretty good code-wise, except for one thing. I made a comment three days ago about the name of this class possibly being too general, and suggesting a renaming. Did you see it? https://github.com/oppia/oppia/pull/3458#pullrequestreview-39602357